exercise.ipynb

```diff
@@ -643,11 +690,14 @@
     "batch_size = 10\n",
     "num_epochs = 1500\n",
     "\n",
+    "# Create the Dataloader associated to the training set. \n",
```

Comment:
Maybe there should be one cell or some more text to explain what the difference between a torch dataset and a torch dataloader is? I know I struggled with this on my first go learning torch so I imagine others might profit from some guidance

Reply:
Makes total sense! I've added that a bit further up in the last commit :)